Source/Core/Core/NetPlayClient.cpp

```diff
@@ -365,14 +368,17 @@ unsigned int NetPlayClient::OnData(sf::Packet& packet)
     u32 cid;
     packet >> cid;
 
-    OnData(m_chunked_data_receive_queue[cid]);
-    m_chunked_data_receive_queue.erase(m_chunked_data_receive_queue.find(cid));
-    m_dialog->HideChunkedProgressDialog();
-
-    sf::Packet complete_packet;
-    complete_packet << static_cast<MessageId>(NP_MSG_CHUNKED_DATA_COMPLETE);
-    complete_packet << cid;
-    Send(complete_packet, CHUNKED_DATA_CHANNEL);
+    if (m_chunked_data_receive_queue.find(cid) != m_chunked_data_receive_queue.end())
```

Comment:
m_chunked_data_receive_queue.count(cid)

Reply:
That is functionally equivalent, and `find` is faster when multiple elements of the same value exist, so I use this method as standard practice.